### ISSUE TITLE
`quarkus.container-image.image` is not mapped correctly into `values.yaml` container image.

### ISSUE THREAD

**Issue by pjgg** [quarkiverse/quarkus-helm#41]
Based on Quarkus documentation `quarkus.container-image.image` represents the entire image string, so If I set something like:

application.properties:

```
quarkus.container-image.image=quay.io/pablo/examples-quarkus-helm:v1
```

I would expect a values.yaml like this one:

```
app:
  image: quay.io/pablo/examples-quarkus-helm:v1
```

However, I am getting the following one:

```
app:
  image: pagonzal/examples-quarkus-helm:1.2.1.Beta3-SNAPSHOT
```

Workaround -> define all image params separately

application.properties:
```
quarkus.container-image.registry=quay.io
quarkus.container-image.group=pablo
quarkus.container-image.name=examples-quarkus-helm
quarkus.container-image.tag=v1
```

**Comment by Sgitario** [comment]
Good catch @pjgg ! 
This issue will be fixed as part of https://github.com/dekorateio/dekorate/pull/1031 in the next version of Dekorate/Quarkus (if merged).

**Comment by paulrobinson** [comment]
I see that https://github.com/dekorateio/dekorate/pull/1031 is now merged. Can this issue be progressed?

**Comment by Sgitario** [comment]
This change is part of Dekorate 2.11.2 and Quarkus main has been recently updated to use this version. So, we need to wait for a new Quarkus release, and then I will confirm this is fixed.

**Comment by pjgg** [comment]
Is working as expected on Quarkus 2.12.0.Final

**Comment by Sgitario** [comment]
It's fixed since Quarkus 2.12.2.Final
